docs/Software Architecture Document/Introduction/index.md

```diff
@@ -20,6 +20,7 @@ Abbreviations used in this document please find in the table below.
 | SEE              | Software Engineering Environment |
 | TTS              | Text To Speech                   |
 | VR               | Voice Recognition                |
+| CAN              | Controller Area Network          |
```

Comment:
I don't see any CAN usage abbreviation in the SAD.
Is it redundant term or some more description is not provided?